.evergreen/buildvariants.yml

```diff
@@ -101,6 +101,9 @@ buildvariants:
 
       - name: test-web-sandbox-firefox
 
+      - name: test-multiple-connections-multiple-connections
```

Comment:
I know this naming ended up being awful and ideally we probably need a separate build variant and what not, but it is just temporary. The normal e2e tests will just become the multiple connection e2e tests soon enough. I'm mindful of how much resources this uses so I'm just running it on ubuntu for now too.

Reply:
But I'm also happy to change it if people feel otherwise.

Reply:
actually let me just change it. It is bothering me :)